services/ircd/inspircd/conf.nix

```diff
@@ -379,19 +379,15 @@ in {
     cmd="PRIVMSG $nickrequired :IDENTIFY $pass";
   }];
   randquote = [{ file=./quotes.txt; }];
-  sslprofile = {
```

Comment:
So this is actually the correct syntax already but would require a restart of the server to deploy so id rather revert to the deprecated syntax for now

Reply:
Is restartOnChange set to false for the inspircd service? If not it's going to restart anyway

Reply:
this is the config that currently deployed it was stashed on the box to avoid a restart. committing it as it has been there so long it can just be pushed to another ircd change